Added a note to page (#9646)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md

You can also use the following macros when the exact volume may vary: `%OSDRIVE%`, `%WINDIR%`, `%SYSTEM32%`.

> >[!NOTE]
> >For others to better understand the WDAC policies that has been deployed, we recommend maintaining separate ALLOW and DENY policies on version 1903 and higher.

Reviewer: Looks like the suggested change got skipped. May we have the standard MarkDown indent marker compatibility spacing?
```suggestion
> [!NOTE]
> For others to better understand the WDAC policies that has been deployed, we recommend maintaining separate ALLOW and DENY policies on version 1903 and higher.
```

> > For others to better understand the WDAC policies that has been deployed, we recommend maintaining separate ALLOW and DENY policies on version 1903 and higher.

Reviewer 2: ```suggestion
> For others to better understand the WDAC policies that have been deployed, we recommend maintaining separate allow and deny policies on Windows 10, version 1903 and later.
```

> >[!NOTE]
> >We recommend maintaining separate ALLOW and DENY policies on version 1903 and higher, if for no other reason than it makes it a bit easier for an average person to reason over the policy.

Reviewer: Please include MarkDown indent marker compatibility spacing:
```suggestion
> [!NOTE]
> We recommend maintaining separate ALLOW and DENY policies on version 1903 and higher, if for no other reason than it makes it a bit easier for an average person to reason over the policy.
```
The last half of the sentence could use some phrasing improvement, but I am uncertain how to phrase it to clarify the intended meaning.
